web/src/pages/Upstream/Create.tsx

```diff
@@ -44,6 +44,14 @@ const Page: React.FC = (props) => {
           }
         }
 
+        if (newData?.checks?.active) {
+          const host = newData?.checks?.active.host;
+          const http_path = newData?.checks?.active.http_path;
+          const url = host + http_path;
+          newData.checks.active.url = url;
```

Comment:
Better to use template literals.

Reply:
means using this syntax
![image](https://user-images.githubusercontent.com/72343596/158722129-0b3f6f32-46b6-41c1-8213-b6aa218dcd0f.png)